packages/btc/src/api/sendRgbppUtxos.ts

```diff
@@ -46,21 +47,45 @@ export async function createSendRgbppUtxosBuilder(props: SendRgbppUtxosProps): P
   const config = networkTypeToConfig(props.source.networkType);
   const isCkbMainnet = props.source.networkType === NetworkType.MAINNET;
 
+  // Batch querying live cells from CkbCollector
+  const ckbLiveCells = await Promise.all(
+    ckbVirtualTx.inputs.map((ckbInput) => {
+      return limitPromiseBatchSize(async () => {
+        const ckbLiveCell = await props.ckbCollector.getLiveCell(ckbInput.previousOutput!);
+        const isRgbppLock = isRgbppLockCell(ckbLiveCell.output, isCkbMainnet);
+        const lockArgs = isRgbppLock ? unpackRgbppLockArgs(ckbLiveCell.output.lock.args) : undefined;
+        return {
+          ckbLiveCell,
+          isRgbppLock,
+          lockArgs,
+        };
+      });
```

Comment:
Does this `ckbCollector RPC` support batch request?

## References
- https://www.jsonrpc.org/specification#batch
- https://github.com/ckb-js/lumos/blob/40e12a95e95e9d0a052a4829679351ee3c6384bf/packages/rpc/src/index.ts#L86

Reply:
The ckbCollector is a wrapped class from the `rgbpp/ckb`, it has no batch request support yet: https://github.com/ckb-cell/rgbpp-sdk/blob/develop/packages/ckb/src/collector/index.ts#L36

Reply:
If necessary,  I will add batch request support to the collector.

Reply:
It's not a high priority, take your time.